remotes/handlers.go

```diff
@@ -243,8 +243,9 @@ func PushContent(ctx context.Context, pusher Pusher, desc ocispec.Descriptor, st
 			// as a marker for this problem
 			if (manifestStack[i].MediaType == ocispec.MediaTypeImageIndex ||
 				manifestStack[i].MediaType == images.MediaTypeDockerSchema2ManifestList) &&
-				errors.Cause(err) != nil && strings.Contains(errors.Cause(err).Error(), "400 Bad Request") {
-				return errors.Wrap(err, "manifest list/index references to blobs and/or manifests are missing in your target registry")
+				// TODO (jonyhy96) veryfi this works as expect
+				errors.Unwrap(err) != nil && strings.Contains(errors.Unwrap(err).Error(), "400 Bad Request") {
```

Comment:
That `if err != nil` condition can guarantee that the change is ok. But your comment is typo

Reply:
Verified this works as espect